model/historical_test_data.go

```diff
@@ -205,7 +201,7 @@ func (d *HistoricalTestData) Remove(ctx context.Context) error {
 		"op":           "remove historical test data record",
 	})
 
-	return errors.Wrapf(err, "problem removing test results record with id %s", d.ID)
+	return errors.Wrapf(err, "removing test results record '%s'", d.ID)
```

Comment:
Just realized this should be `removing historical test data record '%s'`, whoops.

Reply:
Fixed.